### ISSUE TITLE
Can't deploy

### ISSUE THREAD

**Issue by Nuruddinjr** [gdg-x/hoverboard#474]
Can't deploy from latest master branch. 
I run` yarn deploy`, it failed and to reproduce the bug, I run 1. `firebase use devfest18-dev` 2. `firebase deploy`

Here is the log

```
=== Deploying to 'devfest18-dev'...

i  deploying database, firestore, functions, hosting
Running command: npm --prefix $RESOURCE_DIR run build

> hoverboard-functions@ prebuild /Users/nuruddin/Desktop/devfest-website/functions
> rimraf dist


> hoverboard-functions@ build /Users/nuruddin/Desktop/devfest-website/functions
> babel src --out-dir dist --source-maps

🎉  Successfully compiled 7 files with Babel.

> hoverboard-functions@ postbuild /Users/nuruddin/Desktop/devfest-website/functions
> cp ../build/index.html dist/

cp: ../build/index.html: No such file or directory
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! hoverboard-functions@ postbuild: `cp ../build/index.html dist/`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the hoverboard-functions@ postbuild script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/nuruddin/.npm/_logs/2018-07-16T15_11_01_763Z-debug.log

Error: functions predeploy error: Command terminated with non-zero exit code1
```

**Comment by ozasadnyy** [comment]
please use `yarn build` and `firebase deploy` for now.

**Comment by ozasadnyy** [comment]
@Nuruddinjr could you check if you still have the issue after #478 ?

**Comment by Nuruddinjr** [comment]
@ozasadnyy will check it out later tonight.

**Comment by ozasadnyy** [comment]
@Nuruddinjr any news here?

**Comment by abraham** [comment]
Please open a new issue if there are any other issues with deploying.
